app/routes/certificates/instructions.tsx

```diff
@@ -0,0 +1,21 @@
+import { Heading, Text, Link } from '@chakra-ui/react';
```

Comment:
What are the rules about using Chakra UI Link vs. Remix Link?  I wonder if we should prefer the router's link component?

Reply:
You require an `as` tag for when you're using a router library. I'm using Chakra for my other project, and it was suggested I rename `Link` as `ReachLink` for chakra component as implied in their [doc](https://chakra-ui.com/docs/components/link)

Reply:
Oh, I simply used a `Link` here to render a `<a>` element which links to #114